server/src/main/java/org/elasticsearch/index/engine/Engine.java

```diff
@@ -1007,7 +1007,7 @@ protected void writerSegmentStats(SegmentsStats stats) {
         stats.addIndexWriterMemoryInBytes(0);
     }
 
-    /** How much heap is used that would be freed by a refresh.  Note that this may throw {@link AlreadyClosedException}. */
```

Comment:
This did not seem to be an accurate description and I've changed to be similar to what is mentioned in [IndexShard#getIndexBufferRAMBytesUsed](https://github.com/elastic/elasticsearch/blob/main/server/src/main/java/org/elasticsearch/index/shard/IndexShard.java#L2286) which is the only place using this value.

Reply:
I have kept the original comment and just added some detail to it, to indicate that this includes memory being released as well, which is what the PR is adding.